admin/autoupdater/src/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.8.8)
+
+project(AUTOUPDATER C)
+
+set_property(DIRECTORY PROPERTY COMPILE_DEFINITIONS _GNU_SOURCE)
+
+find_path(UBOX_INCLUDE_DIR NAMES libubox/uloop.h)
+find_library(UBOX_LIBRARY NAMES ubox)
+find_library(UCLIENT_LIBRARY NAMES uclient)
+
+find_library(UCI_LIBRARY NAMES uci)
+find_library(PLATFORMINFO_LIBRARY NAMES platforminfo)
```

Comment:
It would be better to write proper CMake package scripts for these libraries instead of using *find_library* directly. See https://gitlab.com/neoraider/ece/tree/master/cmake for a few examples.